<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiscordMovieListForm</class>
 <widget class="QWidget" name="DiscordMovieListForm">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Discord Movie List</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>681</width>
     <height>371</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>681</width>
     <height>371</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>681</width>
     <height>371</height>
    </size>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>679</width>
      <height>369</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="minimumSize">
        <size>
         <width>641</width>
         <height>121</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>641</width>
         <height>121</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="lineWrapMode">
        <enum>QPlainTextEdit::WidgetWidth</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="plainText">
        <string>www.imdb.com/title/tt0076759
www.imdb.com/title/tt0076759/parentalguide
```css
[ Star Wars: Episode IV - A New Hope | ‚åö 121 min | ‚≠ê 8.6/10 | üìÖ 25 May 1977 | PG ]
'Luke Skywalker joins forces with a Jedi Knight, a cocky pilot, a Wookiee and two droids to save the galaxy from the Empire's world-destroying battle station, while also attempting to rescue Princess Leia from the mysterious Darth Vader.'
```</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
